scripts/update-go-version.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
This file has a large overlap with the check script, they need to be in sync regarding all the places they check. It would make sense to combine them into one and have an optional "update" behavior. Eg check script can take -u flag, and update script simply calls 'check -u'